Testreglar/3.3.1/Nett/nett-3.3.1a.json

```diff
@@ -0,0 +1,242 @@
+	"id": "nett-3.3.1a",
+	"testlabId": 410,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Dersom det blir oppdaget automatisk at obligatoriske skjemaelement ikke er fylt ut, er følgende punkter oppfylt:</p>\r\n<ul>\r\n<li>Skjemaelementet der feilen oppstod identifiseres og</li>\r\n<li>Brukeren får en presis tekstlig beskrivelse av feilen og</li>\r\n<li>Feilmeldingen er kodet som tekst </li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "2.6"
+		},
+		{
+			"title": "2.7"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsiden digitale skjema/skjemaelement?",
+			"ht": "<p><strong>Merk:</strong> Du skal ikke teste skjema i PDF, Word eller lignende.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke skjema/skjemaelement."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hvilket skjema/skjemaelement tester du?",
+			"ht": "<p>Beskriv skjema/skjemaelement, slik at det er mulig å identifisere det i ettertid. </p>\n<p><strong>Merk:</strong> Hvis skjema/skjemaelement inngår i en prosess skal du også beskrive prosessen.</p>",
+			"type": "tekst",
+			"label": "Skjema/prosess:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Er det forhåndsutfylt informasjon i skjema?",
+			"ht": "<p>Forhåndsutfylt informasjon betyr at det allerede står inndata i skjemaet når testsiden er lastet inn.</p>\n<p>Eksempel på forhåndsutfylt informasjon, er informasjon hentet fra folkeregisteret som for eksempel personnummer, fornavn, etternavn og adresse.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.5"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "2.7"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Forsøk å fjerne all informasjon i skjema.",
+			"ht": "<p>Forsøk å fjerne all informasjon i alle skjemaelementer i skjema.</p>",
+			"type": "instruksjon",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.6"
+				}
+			}
+		},
+		{
+			"stegnr": "2.6",
+			"spm": "Er det mulig å fjerne forhåndsutfylt informasjon i alle skjemaelementer?",
```

Comment:
Er det nødvendig at du må kunne fjerne informasjon i **alle** skjemaelementer? Holder det ikke med minst et? Viss det tenker jeg du kan hoppe rett fra 2.5 -> 2.7 (Ta bort 2.6)

Reply:
@geirsf Er vel da kanskje ikke vits å ha med 2.5 heller? Det gjentas vel i hjelpeteksten til 2.7?

Reply:
Det holder vel med minst et ja. isåfall enig med Adrian (og GS forsåvidt)

Reply:
Fjernet 2.5 og 2.6 og samtidig har oppdatert hjelpeteksten til 2.7 med informasjon om forhåndsutfylt skjemaelementer.